modules/Bio/EnsEMBL/Variation/DBSQL/LDFeatureContainerAdaptor.pm

```diff
@@ -102,14 +102,42 @@ sub max_snp_distance {
   return $self->{'max_snp_distance'};
 }
 
-sub executable {
-  my $self = shift;
-  $BINARY_FILE = shift if @_;
-  unless( $BINARY_FILE ) {
-    my $binary_name = 'calc_genotypes';
-    ($BINARY_FILE) = grep {-e $_} map {"$_/$binary_name"} split /:/,$ENV{'PATH'};
+=head2 r2
```

Comment:
It was not immediately obvious to me what this meant - I think the method (and const) should be called minimum_r2 or min_r2 to make it clear. Same for dprime.